.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+<!--
```

Comment:
Is the naming case sensitive? According to GitHub docs, file name isn't capitalized. Directory name is.

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository#:~:text=To%20store%20your%20file%20in%20a%20hidden%20directory%2C%20name%20the%20pull%20request%20template%20.github/pull_request_template.md.

Reply:
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates#pull-request-templates
> Pull request template filenames are not case sensitive, and can have an extension such as .md or .txt.
Generally OSS projects use uppercase for these files, including README.md, CONTRIBUTING.md, PULL_REQUEST_TEMPLATE.md, so if we want to unify the case here, I'd actually prefer changing `readme.md` to `README.md`